pkg/client/unversioned/request.go

```diff
@@ -616,11 +647,20 @@ func (r *Request) Stream() (io.ReadCloser, error) {
 	if client == nil {
 		client = http.DefaultClient
 	}
+
+	time.Sleep(time.Duration(backoff.calculateBackoff(url)) * time.Second)
```

Comment:
Don't you want to ensure the value > 0 before sleep, otherwise you're forcing a task switch.